cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` instead of `serializers.Serializer` for the `MovieSerializer`. This will simplify the code by automatically generating fields based on the `Movie` model.